test/app-luatest/http_client_test.lua

```diff
@@ -54,7 +54,7 @@ local function start_server(sock_family, sock_addr)
     return server, url, opts
 end
 
-g.before_all(function(cg)
+g.before_each(function(cg)
```

Comment:
It seems, this change is already in master (02fae15a3adb8ea450ebbe3c250a4846cf1cca69).